hardware_layer/hardware_arduino/include/arduino_thrust.h

```diff
@@ -0,0 +1,25 @@
+#ifndef arduino_thrust_h
```

Comment:
The file name doesn't follow Developer's Guide. It should be in `CameCase`

Reply:
Done